### PR TITLE
Fix source.extension.vsixmanifest FxCopAnalyzers VSIX

### LINKED ISSUE #1253: FxCop extension authored incorrectly
the extension available [here](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2017) is not authored correctly.

The VSIX manifest is allows this extension to be installed on any version of VS that is 2017 RTW or newer.
```XML
<Installation>
  <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,]" />
  <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,]" />
</Installation>
```

following the [docs](https://docs.microsoft.com/en-us/visualstudio/extensibility/vsix-extension-schema-2-0-reference) this should read:
```XML
<Installation>
  <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.3.0,]" />
  <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.3.0,]" />
</Installation>
```

Or perhaps even a more specific version

### PR DESCRIPTION
**Pull request by mavasani**
Fixes #1253

### CODE REVIEW

File: src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest [hunk 1]
```diff
@@ -10,8 +10,7 @@
     <Tags>code analysis; roslyn; analyzer; analyzers; live; fxcop; code quality;</Tags>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,]" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,]" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.5.0,]" />
```
Comment (jmarolf) [inline src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: I do not believe we change the visual studio version non-foundational updates.  So 15.3 or 15.6 are valid but not 15.5

Comment (jmarolf) [reply src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: Actually looking at http://versionof.net/ it does not appear to change the minor version number ever. restricting to 15.4.0 would be `15.0.27004.2002` I do not know what the number for 15.5 will be yet.

Comment (mavasani) [reply src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: Ah, so in that case I will change it to 15.3

Comment (mavasani) [reply src/Microsoft.CodeAnalysis.FxCopAnalyzers/Setup/source.extension.vsixmanifest]: > it does not appear to change the minor version number ever

Thanks, I will close the PR for now and create a separate one once we know the 15.5 version.

**Pull request closed without merging**